integration_tests/test_fee_token.py

```diff
@@ -73,8 +71,8 @@ def kill_processes(parent_id):
 
 
 def check_initialization():
-    assert w3.eth.chain_id == chain_id  # 1337
-    assert w3.eth.block_number == block_number + 1
+    assert w3.eth.chain_id == chain_id
+    assert w3.eth.block_number in [block_number, block_number + 1]
```

Comment:
The block number is deterministic: you should have an equality check here, just get rid of the `+1`